core/nodepool/config/config.go

```diff
@@ -178,6 +178,8 @@ func (c *ProvidedConfig) Load(main *cfg.Config) error {
 	c.KubeClusterSettings = main.KubeClusterSettings
 	c.Experimental.TLSBootstrap = main.DeploymentSettings.Experimental.TLSBootstrap
 	c.Experimental.NodeDrainer = main.DeploymentSettings.Experimental.NodeDrainer
+	c.DeploymentSettings.CloudWatchLogging = main.DeploymentSettings.CloudWatchLogging
+	c.DeploymentSettings.JournaldCloudWatchLogsImage = main.DeploymentSettings.JournaldCloudWatchLogsImage
```

Comment:
Also, I'd suggest that in the `WithDefaultsFrom` func make assignments conditional like we currently do for other fields of DeploymentSettings:

```golang
func (c DeploymentSettings) WithDefaultsFrom(main cfg.DeploymentSettings) DeploymentSettings {
    // snip
    if c.CloudWatchLogging == "" {
        c.CloudWatchLogging = main.CloudWatchLogging
    }
    if c.JournaldCloudWatchLogsImage == "" {
        c.JournaldCloudWatchLogsImage = main.JournaldCloudWatchLogsImage
    }
    // snip
}
```

so that you can provide the user chances to (enable CloudWatchLogging|roll-out the new JournaldCloudWatchLogsImage) only subset of node pools.